src/WebSdk/Web/Sdk/Sdk.targets

```diff
@@ -22,6 +22,9 @@ Copyright (c) .NET Foundation. All rights reserved.
       Microsoft.AspNetCore.App. This needs to happen after the .NET SDK has evaluated TFMs.
      -->
     <AddRazorSupportForMvc Condition="'$(DisableImplicitFrameworkReferences)' != 'true' And '$(AddRazorSupportForMvc)' == '' And '$(TargetFrameworkIdentifier)' == '.NETCoreApp' And '$(_TargetFrameworkVersionWithoutV)' >= '3.0'">true</AddRazorSupportForMvc>
+
+    <!-- Trimmer defaults that depend on user-definable settings -->
+    <SuppressTrimAnalysisWarnings Condition="'$(SuppressTrimAnalysisWarnings)' == '' And '$(TrimmerDefaultAction)' != 'link'">true</SuppressTrimAnalysisWarnings>
```

Comment:
So ASP.NET has warnings suppressed, but no top-level warning about trimmability? Hmm. We should think about this for .NET 6.